tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3604 @@
+        outputCommitmentProof:
+          description: The proof of the output commitment. Hex-encoded with 0x prefix.
+          anyOf:
+              - $ref: '#/components/schemas/HashableNode'
+              - $ref: '#/components/schemas/LeafHash'
+              - $ref: '#/components/schemas/ValueHash'
+      required:
+        - slot
+        - outputIndex
+        - transactionCommitment
+        - outputCommitmentProof
+
+    HashableNode:
+      description: Node contains the hashes of the left and right children of a node in the tree.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote a Hashable Node.
+        node:
+          properties:
+            l:
+              description: The left node. Hex-encoded with 0x prefix.
+              anyOf:
+              - $ref: '#/components/schemas/HashableNode'
+              - $ref: '#/components/schemas/LeafHash'
+              - $ref: '#/components/schemas/ValueHash'
+            r:
+              description: The right node. Hex-encoded with 0x prefix.
+              anyOf:
+              - $ref: '#/components/schemas/HashableNode'
+              - $ref: '#/components/schemas/LeafHash'
+              - $ref: '#/components/schemas/ValueHash'
+      required:
+        - type
+        - node
+
+    LeafHash:
+      description: Leaf Hash contains the hash of a leaf in the tree.
+      properties:
+        type:
+          type: integer
+          description: Set to value 1 to denote a Leaf Hash.
+        hash:
+          type: string
+          description: The hash of a leaf in the tree. Hex-encoded with 0x prefix.
+      required:
+        - type
+        - hash
+
+    ValueHash:
+      description: Value Hash contains the hash of the value for which the proof is being computed.
+      properties:
+        type:
+          type: integer
+          description: Set to value 2 to denote a Value Hash.
+        hash:
+          type: string
+          description: the hash of the value for which the proof is being computed. Hex-encoded with 0x prefix.
+      required:
+        - type
+        - hash
+
+
+    OutputWithMetadataResponse:
+      description: Returns an output.
+      properties:
+        output:
+          anyOf:
+            - $ref: '#/components/schemas/BasicOutput'
+            - $ref: '#/components/schemas/AccountOutput'
+            - $ref: '#/components/schemas/AnchorOutput'
+            - $ref: '#/components/schemas/FoundryOutput'
+            - $ref: '#/components/schemas/NFTOutput'
+            - $ref: '#/components/schemas/DelegationOutput'
+        outputIdProof:
+          allOf:
+              - $ref: '#/components/schemas/OutputIdProof'
+        metadata:
+          allOf:
+            - $ref: '#/components/schemas/OutputMetadataResponse'
+      required:
+        - output
+        - outputIdProof
+        - metadata
+
+    OutputMetadataResponse:
```

Comment:
Outdated: https://github.com/iotaledger/iota.go/pull/625/files
But the transaction id in the `OutputInclusionMetadata` shouldn't be needed, because it's already part of the  `OutputID` in the `OutputMetadata` (@DaughterOfMars noticed this)

Reply:
We think it is better to have it duplicated again, so no need to cut the outputID manually to get to the txID.